omero/sysadmins/version-requirements.txt

```diff
@@ -256,62 +168,54 @@ Microsoft Windows
       - Release date
       - (mainline)
       - (extended)
-      - OMERO 5.1
       - OMERO 5.2
       - OMERO 5.3
+      - OMERO 5.4
       - Details
-    * - Vista
-      - from Jan 2007
-      - to Apr 2012
-      - to Apr 2017
-      - Dropped
-      - Dropped
-      - Dropped
-      - `Ref <https://support.microsoft.com/en-gb/help/13853>`__
     * - Win 7
       - from Oct 2009
       - to Jan 2015
       - to Jan 2020
-      - Recommended
       - Supported
       - Unsupported†
-      - `Ref <https://support.microsoft.com/en-gb/help/13853>`__
+      - Unsupported†
```

Comment:
Hmmm, what about leaving it at the same URL but removing it from the main sysadmin index and listing it nested under https://www.openmicroscopy.org/site/support/omero/developers/installation.html instead?